src/OrbitGl/TimelineUi.h

```diff
@@ -14,6 +14,8 @@ namespace orbit_gl {
 // TimelineUi is the class which takes care of drawing the timeline in the CaptureWindows.
 class TimelineUi : public CaptureViewElement {
  public:
+  const float kLabelMarginRight = 2;
+  const float kLabelMarginLeft = 4;
```

Comment:
Is there a reason for putting them into the header? And could you make them `constexpr`?

Reply:
Perfect. Thx!